app/controllers/conference_controller.rb

```diff
@@ -14,6 +14,7 @@ def show; end
   def schedule
     @rooms = @conference.venue.rooms if @conference.venue
     @events = @conference.program.events
+    @events_xml = @conference.program.events.scheduled.group_by{ |event| event.start_time.to_date }
```

Comment:
This will probably be cached by the query caching but why do you load everything twice?